src/components/item/publish/CCLicenseDialog.jsx

```diff
@@ -11,6 +9,8 @@ import {
 import { BUILDER } from '@graasp/translations';
 import { Button } from '@graasp/ui';
 
+import PropTypes from 'prop-types';
```

Comment:
Remove? Not a tsx file?

Reply:
It is actually a `jsx` file (vite requires this extension for files written in JS that contain JSX). But I will convert it to `tsx`.